app/src/test/java/com/duckduckgo/app/browser/omnibar/OmnibarLayoutViewModelTest.kt

```diff
@@ -865,11 +865,12 @@ class OmnibarLayoutViewModelTest {
     fun whenTrackersAnimationStartedAndOmnibarFocusedThenCommandAndViewStateCorrect() = runTest {
         testee.onOmnibarFocusChanged(true, SERP_URL)
         val trackers = givenSomeTrackers()
+
         testee.onAnimationStarted(Decoration.LaunchTrackersAnimation(trackers))
 
         testee.viewState.test {
             val viewState = awaitItem()
-            assertTrue(viewState.leadingIconState == LeadingIconState.SEARCH)
+            assertTrue(viewState.leadingIconState == LeadingIconState.PRIVACY_SHIELD)
```

Comment:
This change looks incorrect - if the omnibar is focused, we shouldn't be showing trackers animation, should we?

Reply:
fixed!